frontend/src/components/floating-menus/MenuList.vue

```diff
@@ -20,11 +20,11 @@
 					v-for="(entry, entryIndex) in virtualScrollingEntryHeight ? section.slice(virtualScrollingStartIndex, virtualScrollingEndIndex) : section"
 					:key="entryIndex + (virtualScrollingEntryHeight ? virtualScrollingStartIndex : 0)"
 					class="row"
-					:class="{ open: isEntryOpen(entry), active: entry.label === highlighted?.label }"
+					:class="{ open: isEntryOpen(entry), active: entry.label === highlighted?.label, disabled: entry.disabled }"
 					:style="{ height: virtualScrollingEntryHeight || '20px' }"
-					@click="() => onEntryClick(entry)"
-					@pointerenter="() => onEntryPointerEnter(entry)"
-					@pointerleave="() => onEntryPointerLeave(entry)"
+					@click="() => entry.disabled || onEntryClick(entry)"
+					@pointerenter="() => entry.disabled || onEntryPointerEnter(entry)"
+					@pointerleave="() => entry.disabled || onEntryPointerLeave(entry)"
```

Comment:
I think you want `!entry.disabled && ...` since, if `disabled` is true, you're giving the value of `true` to these event handlers which it might not know what to do with. I think it needs a falsy value, not `true`.

Reply:
I'm not sure why it matters, event handlers can have any return type and it is just discarded by the browser.

Reply:
I was just thinking it might subtly cause warnings from Vue, now or in the future.

Reply:
That warning would be pretty useless and probably spam everyone's projects but I will change it if you think it would be easier to understand?

Reply:
I haven't tested if Vue gives a warning or not. But at least I think it's slightly better practice to make sure the handler gets either a callback or a falsy value, for code consistency with the rest of the codebase at least. Probably not a big deal though. Thanks.